x-pack/legacy/plugins/index_management/public/app/components/mappings_editor/constants/parameters_definition.tsx

```diff
@@ -252,6 +252,16 @@ export const PARAMETERS_DEFINITION = {
       label: nullValueLabel,
     },
   },
+  null_value_ip: {
+    fieldConfig: {
+      defaultValue: '',
+      type: FIELD_TYPES.TEXT,
+      label: nullValueLabel,
+      helpText: i18n.translate('xpack.idxMgmt.mappingsEditor.parameters.nullValueHelpText', {
```

Comment:
Nit: I know we don't have this key in use elsewhere, but maybe it should be `nullValueIpHelpText` to be more specific?

Reply:
👍 good catch